maestro-driver-ios/maestro-driver-iosUITests/Routes/Models/TouchRequest.swift

```diff
@@ -1,4 +1,5 @@
 struct TouchRequest : Codable {
     let x: Float
     let y: Float
+    let durationInMs: Int?
```

Comment:
As a datatype, please use TimeInterval which is a Double value, in seconds.
For the name, I think `pressDuration` would be clearer.

Reply:
Agreed! 
https://github.com/mobile-dev-inc/maestro/pull/905/commits/9ad0a4760d0ef183931b7ef98ae3f3e12d96bc26: I decided to go only for `duration` because the `swipe` request uses `duration` in seconds as well.